src/array/reducer.ts

```diff
@@ -6,7 +6,7 @@ import {
   FocusAction,
   isNgrxFormsAction,
   RemoveGroupControlAction,
-  UnfocusAction,
+  UnfocusAction
```

Comment:
Minor thing, but there's a trailing comma missing here. Not sure why TSLint didn't pick up on it though.